CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+_This is a suggested `CONTRIBUTING.md` file template for use by open sourced Salesforce projects. The main goal of this file is to make clear the intents and expectations that end-users may have regarding this project and how/if to engage with it. Adjust as needed (especially look for `{project_slug}` which refers to the org and repo name of your project) and remove this paragraph before committing to your repo._
+
+# Contributing Guide For tough-cookie
+
+This page lists the operational governance model of this project, as well as the recommendations and requirements for how to best contribute to tough-cookie. We strive to obey these as best as possible. As always, thanks for contributing – we hope these guidelines make it easier and shed some light on our approach and processes.
+
+## Governance Model
+
+The intent and goal of open sourcing this project is to increase the contributor and user base. The governance model is one where new project leads (`admins`) will be added to the project based on their contributions and efforts, a so-called "do-acracy" or "meritocracy" similar to that used by all Apache Software Foundation projects.
+
+## Issues, requests & ideas
+
+Use GitHub Issues page to submit issues, enhancement requests and discuss ideas. Please ensure your description is clear and has sufficient instructions to be able to reproduce the issue.
+
+### Bug Reports and Fixes
+
+- If you find a bug, please search for it in the [Issues](https://github.com/salesforce/tough-cookie/issues), and if it isn't already tracked,
+  [create a new issue](https://github.com/salesforce/tough-cookie/issues/new). Fill out the "Bug Report" section of the issue template. Even if an Issue is closed, feel free to comment and add details, it will still
+  be reviewed.
+- Issues that have already been identified as a bug (note: able to reproduce) will be labelled `bug`.
+- If you'd like to submit a fix for a bug, [send a Pull Request](#creating_a_pull_request) and mention the Issue number.
+- Include tests that isolate the bug and verifies that it was fixed.
+
+### New Features
+
+- If you'd like to add new functionality to this project, describe the problem you want to solve in a [new Issue](https://github.com/salesforce/tough-cookie/issues/new).
+- Issues that have been identified as a feature request will be labelled `enhancement`.
```

Comment:
Our [existing labels](https://github.com/salesforce/tough-cookie/labels) has `FeatureRequest`. We'll either need to create an `enhancement` label or change this text to `FeatureRequest`.

Reply:
```suggestion
- Issues that have been identified as a feature request will be labelled `FeatureRequest`.
```